esp/esp/program/modules/handlers/nametagmodule.py

```diff
@@ -80,14 +82,22 @@ def nametag_data(self, users_list1, user_title1, users_list2 = ESPUser.objects.n
         users_list.sort()
 
         for user in users_list:
+            prof = RegistrationProfile.getLastProfile(user)
             if user in users_list1:
                 title = user_title1
             else:
                 title = user_title2
-            user_dict = {'title': title,
-                         'name' : '%s %s' % (user.first_name, user.last_name),
-                         'id'   : user.id,
-                         'username': user.username}
+            if prof.student_info is not None:
+                user_dict = {'title': title,
+                             'name' : '%s %s' % (user.first_name, user.last_name),
+                             'id'   : user.id,
+                             'username': user.username,
+                             'pronoun': prof.student_info.pronoun}
+            else:
+                user_dict = {'title': title,
+                             'name' : '%s %s' % (user.first_name, user.last_name),
+                             'id'   : user.id,
+                             'username': user.username}
```

Comment:
This code looks a little strange. Why does `prof.student_info is not None` determine whether to display pronouns or not ?

Reply:
I believe that calling prof.student_info.pronoun yielded an error if prof.student_info was None, so I had to make sure it existed first

Reply:
of course, sorry. i missed that it was prof.student_info.pronoun.